README.md

```diff
@@ -190,20 +192,115 @@ Shortcut | Description
 `Super`+`Up` | Maximize
 `Super`+`Down` | Un-Maximize (return to size and position previous to maximizing)
 `Super`+`Left/Right` | left/right half of screen
-`Shift`+`Super`+`Up/Down/Left/Right` | move window to adjacent monitor/workspace 
+`Shift`+`Super`+`Up/Down/Left/Right` | move window to adjacent monitor/workspace
 
+# Developer Documentation
```

Comment:
This PR is difficult to review well because it is so large.  Prefer smaller pull requests [for reasons discussed here](https://www.swarmia.com/blog/why-small-pull-requests-are-better/) and discussing major rewrites like this beforehand. The change of build system / release scripts and deletion of test cases are things I would have pushed back on early on.

Reply:
Do you have a suggestion on what  I could  have made differently, considering that its in the nature of a rewrite to be large (and essentially replacing the old code base in its entirety)?

Reply:
> Do you have a suggestion on what  I could  have made different, considering that its in the nature of a rewrite to be large (and essentially replacing the old code base in its entirety)?

Stacked commits are great for this sort of thing. There are a few open source solutions, as well as productized solutions (like Graphite).

That said, stacking is a bit of a shift compared to how we're used to dealing with git/PRs on GitHub — so it's a bit of a learning curve.